go/src/lib.rs

```diff
@@ -379,6 +398,36 @@ pub unsafe extern "C" fn command(
                 command_response.float_value = num;
                 Ok(Some(command_response))
             }
+            Value::Boolean(boolean) => {
+                command_response.bool_value = boolean;
+                Ok(Some(command_response))
+            }
+            Value::Array(array) => {
+                let mut vec_elements_len: Vec<c_long> = Vec::new();
+                let vec = array
+                    .iter()
+                    .map(|value| {
+                        if *value == Value::Nil {
+                            vec_elements_len.push(0);
+                            return std::ptr::null_mut();
+                        }
+                        let res = <String>::from_redis_value(value)
```

Comment:
why are we assuming that all elements are of type `String` here?

Reply:
Because currently we are only handling the case of *mut *mut c_char, we will handle other cases as well as we implement more commands and encounter other use cases.

Reply:
I'd rather fix the binary support now before implementing more commands

Reply:
I have modified the code to use Vec<u8> for now. I am planning to make this more generic to handle all types of arrays in the upcoming PRs.